netty/src/main/java/io/grpc/netty/NettyChannelBuilder.java

```diff
@@ -129,6 +134,16 @@ public NettyChannelBuilder flowControlWindow(int flowControlWindow) {
     return this;
   }
 
+  /**
+   * Sets the maximum message size allowed to be received on the channel. If not called,
+   * defaults to 100 MiB.
```

Comment:
Could the comment reference DEFAULT_MAX_MESSAGE_SIZE instead of hard-coding 100 MiB?